CMakeLists.txt

```diff
@@ -294,6 +294,14 @@ endif()
 
 option(UA_FORCE_WERROR "Force compilation with -Werror" ON)
 
+option(UA_ENABLE_ENCRYPTION_TPM_KEYSTORE "Enable TPM hardware for key encryption" OFF)
```

Comment:
Describe the flag in the build documentation.

say explicitly, that this feature adds examples and tools.
But does not change the core library.